symengine/lib/symengine_wrapper.pyx

```diff
@@ -1482,7 +1602,7 @@ class NegativeInfinity(Number):
         import sage.all as sage
         return -sage.oo
 
-class ComplexInfinity(Number):
+class ComplexInfinity(with_metaclass(Singleton, Number)):
```

Comment:
This code is essentially a mirror of SymPy's `Singleton` class. Though this fails at the above assertion.